cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -526,20 +526,25 @@ define([
                 expect(this.view.$el).toContainText(emptyMessage);
                 expect(this.view.$(SELECTORS.itemView)).not.toExist();
             });
+
+            it('can create a new group configuration', function () {
+                this.view.$('.new-button').click();
+                expect($('.group-configuration-edit').length).toBeGreaterThan(0);
+            });
         });
     });
 
-    describe('GroupConfigurationItem', function() {
+    describe('GroupConfigurationItemView, the controller view for experiment group configurations', function() {
```

Comment:
I was implementing the tests using the style you suggest (`describe`s specify just the name of the object being tested), but @cahrens found them difficult to read.. not sure how to resolve this disagreement.